packages/react-native-sdk/src/utils/push/android.ts

```diff
@@ -169,6 +169,36 @@ export const firebaseDataHandler = async (
     const created_by_id = data.created_by_id as string;
     const receiver_id = data.receiver_id as string;
 
+    const shouldRejectCallWhenBusy = pushConfig.shouldRejectCallWhenBusy;
+
+    const video_client = await pushConfig.createStreamVideoClient();
```

Comment:
Are we sure this won't create multiple client instances in case an integrator implements this method with `new StreamVideoClient()`?
Can we somehow shield ourselves from that?